schema/data/portal/application.sql

```diff
@@ -3,7 +3,7 @@ begin;
 with rows as (
 insert into ggircs_portal.application(id, facility_id, reporting_year)
 overriding system value
-values (1,3, 2018), (2,2, 2019)
+values (1,3, 2018)
```

Comment:
This was causing a duplicate because of a function call `select create_application_mutation_chain(2)` in schema/data/portal/form_result.sql. Since we were hardcoding (2,2,2019) and making this call it was making 2 applications for the facility with id=2. We handled the problem caused by having two applications for one facility by making the facilities search function more robust, but we probably shouldn't be making duplicate applications in the dummy data anyway.